README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
-| `show` for counterexample output                                                          | 🚧      | ✔️                                                           | ✔️                                                                |
-| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | ✖️                                                                |
+| `ppShow` instead of `show` for counterexample output                                      | ✔️       | ✖️                                                           | C                                                                |
+| `show` for counterexample output                                                          | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hunit)           |
+| Fancy Unicode output                                                                      | ✔️       | ✖️                                                           | C                                                                |
 | Unicode-free output                                                                       | 🚧      | ✖️                                                           | ✔️                                                                |
-| Assertion-specific output with explanation                                                | ✔️       | ✔️                                                           | ?                                                                |
-| Coloured diffing                                                                          | ✔️       | ✔️                                                           | ✖️                                                                |
+| Assertion-specific output with explanation                                                | ✔️       | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hedgehog)        |
+| Coloured diffing                                                                          | ✔️       | ✔️                                                           | C                                                                |
 | multi-line diffing                                                                        | ✔️       | ✖️                                                           | ✖️                                                                |
 | Contextual failures                                                                       | ✔️       | ✖️                                                           | ✖️                                                                |
```

Comment:
Contextual failures is where you may have multiple assertions in your test, and when the test fails you see which one it was in the output.
For example
```
context 1 $ a `shouldBe` 400
context 2 $ b `shouldBe` 400
```

Then you see "actual: 300, expected: 400" but you don't know which assertion it was that failed.